src/common/options.cc

```diff
@@ -4538,7 +4538,7 @@ std::vector<Option> get_global_options() {
     .set_description("Enable use of rocksdb column families for bluestore metadata"),
 
     Option("bluestore_rocksdb_cfs", Option::TYPE_STR, Option::LEVEL_DEV)
-    .set_default("m(3) O(3,0-13) L")
+    .set_default("m(3) p(3) O(3,0-13) L")
```

Comment:
"p(3,0-12)" is better. 
It will make hash calculation for column family sharding purposes to bytes 0-11.
I just assume that it is better to keep all omaps for one pg in single rocksdb column.